pkg/utils/posthog.go

```diff
@@ -1,19 +1,27 @@
 package utils
 
 import (
+	"fmt"
+
 	"github.com/posthog/posthog-go"
 )
 
 var (
 	writeOnlyPublicPosthogKey = "dev"
-	posthogEndpoint           = "https://app.posthog.com"
+	posthogEndpoint           = "https://us.i.posthog.com"
 )
 
 // PosthogCliClient is a wrapper around the posthog-go client and is used as a
 // API entrypoint for sending OpenSauced telemetry data for CLI commands
 type PosthogCliClient struct {
-	client    posthog.Client
+	// client is the Posthog Go client
+	client posthog.Client
+
+	// activated denotes if the user has turned off telemetry
```

Comment:
nit (wording): Also, consider mentioning that by default it's enabled.

```suggestion
	// activated denotes if the user has enabled or disabled telemetry
```